tests/slack_bolt/cli/test_get_manifest.py

```diff
@@ -0,0 +1,24 @@
+import json
+import pytest
+
+from slack_bolt.cli.get_manifest import get_manifest
+
+
+class TestGetManifest:
+    def test_get_manifest(self):
+        working_directory = "tests/slack_bolt/cli/test_directory"
```

Comment:
test_directory sounds like a place where we may create and delete files as a sandbox. If I understand correctly, the files under the directory looks like example apps used for the testing. How about renaming to sample/example_app or test_app?

Reply:
I've renamed the folder to test_app, thanks